src/hammer-vlsi/defaults.yml

```diff
@@ -608,3 +619,51 @@ technology.core:
     # Can be a wildcard/glob
     # type: str
     tap_cell_rail_reference: null
+
+sim.inputs:
+  # Top RTL module
+  top_module: null
+  # top_module_config: null
+  input_files: []
+  # Debug option
+  debug: false
+  # Black box command line option escape hatch
+  options: [] 
```

Comment:
what does this do?

Reply:
More comments were added to the defaults.yml since.  The debug flag enable vcd generation and the options flag is for general black box options that will be passed directly as a flag when compiling the test executable as an exact string.